packages/crypto-server/src/@types/types.ts

```diff
@@ -4,16 +4,16 @@
 };
 
 export type IHeadersGenerate = {
-  type: string;
-  bits: string;
-  name: string;
+  curve: any;
+  date: Date;
   email: string;
+  format: any;
+  keyExpirationTime: number;
+  name: string;
   passphrase: string;
-  curve: string;
-  expiration: string;
-  format: string;
-  message: string;
-  sign: string;
+  rsaBits: number;
+  type: any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/sebastienrousseau/crypto-service/pullRequest?prid=9768072)
```suggestion
  type: unknown;
```